src/App.js

```diff
@@ -45,10 +137,9 @@ const removeTodo = (id) => {
 }
   return (
     <>
-    {isLoading?<p>...isloading</p>:<p></p>}
-      <h1>Todo List</h1>
+    <h1>Todo List</h1>
       <AddTodoForm onAddTodo={addTodo}/>
-      <TodoList todoList={todoList} onRemoveTodo={removeTodo}/>
+       {isLoading?<p>...isloading</p>: <TodoList todoList={todoList} onRemoveTodo={removeTodo}/>}
```

Comment:
For the paragraph you render when the app is still loading, we can put any text we want - it doesn't have to match the `isLoading` conditional. 

Maybe something like

```suggestion
       {isLoading?<p>Loading...</p>: <TodoList todoList={todoList} onRemoveTodo={removeTodo}/>}
```

Reply:
Fixed